### ISSUE TITLE
Upgrading minikube failed with tag SHA error

### ISSUE THREAD

**Issue by devhammed** [Homebrew/homebrew-core#60746]
# Bug report

**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [x] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link? https://pastebin.com/iDkM9qGW
- [x] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue? https://pastebin.com/9j6Yhinf

## What you were trying to do (and why)

Trying to upgrade my packages.

## What happened (include command output)

> Error: v1.13.0 tag should be eeb05350f8ba6ff3a12791fcce350c131cb2ff44
but is actually 0c5e9de4ca6f9c55147ae7f90af97eff5befef5f.

![Screenshot from 2020-09-07 11-08-53](https://user-images.githubusercontent.com/22827908/92376630-c7097f00-f0fa-11ea-8968-4a35bd7a9ded.png)

## What you expected to happen

Upgrade.

## Step-by-step reproduction instructions (by running `brew install` commands)

You need to have `minikube 1.12.3` installed, then run `brew upgrade`.

**Comment by SMillerDev** [comment]
It seems that minikube retagged. I asked if that was intentional in: https://github.com/kubernetes/minikube/issues/9196

**Comment by devhammed** [comment]
@SMillerDev Alright, I will be watching the issue too.

**Comment by pcallahan-r7** [comment]
maybe related?

```
$ brew uninstall minikube

Error: undefined method `name' for nil:NilClass
Please report this issue:
  https://docs.brew.sh/Troubleshooting
/usr/local/Homebrew/Library/Homebrew/cli/named_args.rb:184:in `warn_if_cask_conflicts'
/usr/local/Homebrew/Library/Homebrew/cli/named_args.rb:105:in `block in to_kegs_to_casks'
/usr/local/Homebrew/Library/Homebrew/cli/named_args.rb:102:in `each'
/usr/local/Homebrew/Library/Homebrew/cli/named_args.rb:102:in `to_kegs_to_casks'
/usr/local/Homebrew/Library/Homebrew/cmd/uninstall.rb:52:in `uninstall'
/usr/local/Homebrew/Library/Homebrew/brew.rb:113:in `<main>'
```

**Comment by SMillerDev** [comment]
That's not related. That's an issue in @Homebrew/brew

**Comment by fstokke** [comment]
I get the same error. The problem also seems to occur on a fresh install (not upgrade) of minikube 1.13.0.

**Comment by SMillerDev** [comment]
And it'll keep happening for everyone building from source until we're sure that the retag was intentional.

**Comment by denismoc** [comment]
Any update on this? I'm having the same issue when performing a fresh install.

**Comment by chenrui333** [comment]
I will try to tackle this after go1.15.2 merge.

**Comment by chenrui333** [comment]
Should be fixed now. @devhammed

**Comment by devhammed** [comment]
@chenrui333 It is still happening.

**Comment by devhammed** [comment]
I think they revert the tagging and the reverse of above is now happening:

```
==> Downloading https://linuxbrew.bintray.com/bottles/go-bindata-3.21.0.x86_64_linux.bottle.tar.gz
Already downloaded: /home/devhammed/.cache/Homebrew/downloads/55706724bcb823f0fb9fa2de4b02eada4fe361a0cd0cd57a5ae13172984cb57f--go-bindata-3.21.0.x86_64_linux.bottle.tar.gz
==> Cloning https://github.com/kubernetes/minikube.git
Updating /home/devhammed/.cache/Homebrew/minikube--git
==> Checking out tag v1.13.0
HEAD is now at 0c5e9de Merge pull request #9182 from sharifelgamal/fix-release-make
HEAD is now at 0c5e9de Merge pull request #9182 from sharifelgamal/fix-release-make
Entering 'site/themes/docsy'
Synchronizing submodule url for 'assets/vendor/Font-Awesome'
Synchronizing submodule url for 'assets/vendor/bootstrap'
Entering 'site/themes/docsy/assets/vendor/Font-Awesome'
Entering 'site/themes/docsy/assets/vendor/bootstrap'
/home/devhammed/.cache/Homebrew/minikube--git/site/themes/docsy
/home/devhammed/.cache/Homebrew/minikube--git/site/themes/docsy/assets/vendor/Font-Awesome
/home/devhammed/.cache/Homebrew/minikube--git/site/themes/docsy/assets/vendor/bootstrap
Error: v1.13.0 tag should be eeb05350f8ba6ff3a12791fcce350c131cb2ff44
but is actually 0c5e9de4ca6f9c55147ae7f90af97eff5befef5f
```

**Comment by chenrui333** [comment]
I dont have problem.

```
$ brew reinstall -s minikube
==> Cloning https://github.com/kubernetes/minikube.git
Updating /Users/rchen/Library/Caches/Homebrew/minikube--git
==> Checking out tag v1.13.0
HEAD is now at 0c5e9de Merge pull request #9182 from sharifelgamal/fix-release-make
HEAD is now at 0c5e9de Merge pull request #9182 from sharifelgamal/fix-release-make
Entering 'site/themes/docsy'
Synchronizing submodule url for 'assets/vendor/Font-Awesome'
Synchronizing submodule url for 'assets/vendor/bootstrap'
Entering 'site/themes/docsy/assets/vendor/Font-Awesome'
Entering 'site/themes/docsy/assets/vendor/bootstrap'
/Users/rchen/Library/Caches/Homebrew/minikube--git/site/themes/docsy
/Users/rchen/Library/Caches/Homebrew/minikube--git/site/themes/docsy/assets/vendor/Font-Awesome
/Users/rchen/Library/Caches/Homebrew/minikube--git/site/themes/docsy/assets/vendor/bootstrap
```

**Comment by devhammed** [comment]
I think you fixed for MaOS while I am on Linux, I have opened a pull request on linuxbrew repository that fixes the issue: https://github.com/Homebrew/linuxbrew-core/pull/21102

**Comment by Rylan12** [comment]
@devhammed this change will be merged into linuxbrew-core manually by one of the Linux maintainers (usually around once a day). See the [linuxbrew contribution guidelines](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md#contributing-to-linuxbrew).

**Comment by waelkdouh** [comment]
Just tried to install it and I am getting the same error
Error: v1.13.0 tag should be eeb05350f8ba6ff3a12791fcce350c131cb2ff44
but is actually 0c5e9de4ca6f9c55147ae7f90af97eff5befef5f

**Comment by SMillerDev** [comment]
On linux?

**Comment by waelkdouh** [comment]
I am using WSL 2 on Windows 10. the distro I am using is Ubuntu 20.04

**Comment by waelkdouh** [comment]
> On linux?

I am using WSL 2 on Windows 10. the distro I am using is Ubuntu 20.04

**Comment by SMillerDev** [comment]
Then you'll have to wait until it's merged into linuxbrew.

**Comment by iMichka** [comment]
Fixed with https://github.com/Homebrew/linuxbrew-core/pull/21103

**Comment by waelkdouh** [comment]
> Fixed with [Homebrew/linuxbrew-core#21103](https://github.com/Homebrew/linuxbrew-core/pull/21103)

Thanks Michka. Just tried it and works like a charm.
